nixos/services/ceph/client.nix

```
@@ -152,6 +150,32 @@ in
             Can override existing default setting values. Configuration keys like `mon osd full ratio` can alternatively be written in camelCase as `monOsdFullRatio`.
             '';
         };
+
+        mons = lib.mkOption {
+          type = with lib.types; listOf str;
+          default = (map
+              (mon: "${head (lib.splitString "." mon.address)}.${cfg.network.vlan}.${location}.ipv4.gocept.net")
```

doesn't eval, need to be

```suggestion
              (mon: "${head (lib.splitString "." mon.address)}.${cfg.client.network.vlan}.${location}.ipv4.gocept.net")
```